src/app/(proper_react)/(onboarding)/user/welcome/Onboarding.test.tsx

```diff
@@ -33,7 +33,7 @@ it("can open a dialog with more information", async () => {
   expect(screen.queryByRole("dialog")).not.toBeInTheDocument();
 
   const explainerTrigger = screen.getByRole("button", {
-    name: "onboarding-get-started-content-explainer",
+    name: "See how we protect your data",
```

Comment:
Yeah, it would. (That was already the case, though, as string changes would also require a new l10n ID.) [The idea](https://testing-library.com/docs/queries/about#priority) is that we simulate what the user would do, so that e.g. if a "Sign up" button were to suddenly say "Sign in", the tests break because the user's expected flow would break.

It's not that much work to fix since the test runner should clearly indicate which tests fail, and strings don't change that often, so IMHO that's a reasonable trade-off. (But: definitely something we could discuss in the tech meeting if you're interested.)